cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -49,6 +55,113 @@ subjects:
   name: rook-ceph-cluster
   namespace: rook-ceph
 ---
+kind: ClusterRoleBinding
```

Comment:
Can we declare this as a `RoleBinding` instead of `ClusterRoleBinding`? We would like to keep everything cluster-specific in a namespace.

Reply:
Could you add a comment such as [this](https://github.com/rook/rook/pull/2001/files#diff-45b3505b4ed0aaf94f47dc2161c6cc5dR43) above each role binding to state its purpose? I have to think too hard every time I look at the bindings unless there is a description.